horoscope/horoscope.py

```diff
@@ -212,59 +219,23 @@ async def _cookie(self, ctx):
             info = re.findall(regex[4], test)
             info[0] = html.unescape(info[0])
             dailynum = re.findall(regex[5], test)
-            self.fortune_process(fortune[0])
-            await ctx.send("Your fortune is:", file=discord.File(str(bundled_data_path(self) / "cookie-edit.png")))
+            img_png = self.fortune_process(fortune[0])
```

Comment:
Shouldn't there be a failsafe function for the cookie.png if for any weird reason the cookie file goes kapoof now that file_check fn is gone.